<?xml version="1.0"?>
<robot name="box_bot">

  <material name="red">
      <color rgba="1.0 0.0 0.0 1"/>
  </material>

  <material name="green_light">
      <color rgba="0.0 1.0 0.0 1"/>
  </material>

  <material name="green_dark">
    <color rgba="0.0 0.5 0.0 1"/>
  </material>

  <material name="blue">
      <color rgba="0.0 0.0 1.0 1"/>
  </material>

  <material name="yellow">
      <color rgba="1.0 1.0 0.0 1"/>
  </material>

  <material name="black">
      <color rgba="0.0 0.0 0.0 1"/>
  </material>

  <material name="white">
      <color rgba="1.0 1.0 1.0 1"/>
  </material>

  <material name="box_bot_blue">
      <color rgba="0.088656 0.428691 0.491021 1"/>
  </material>

  <link name="base_link">
  </link>


  <!-- Body -->
  <link name="chassis">
    <visual>
      <geometry>
        <mesh filename="package://main_pkg/meshes/cube_tag3_blue.dae" scale="0.025 0.025 0.025"/>
      </geometry>
      <material name="box_bot_blue"/>
    </visual>

    <collision>
      <geometry>
        <box size="0.05 0.05 0.05"/>
      </geometry>


    </collision>

    <inertial>
      <mass value="0.1"/>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <inertia ixx="0.0000415" ixy="0" ixz="0" iyy="0.0000415" iyz="0" izz="0.0000415"/>  <!-- for a box: ixx = 0.083 * mass * (y*y + z*z) -->
    </inertial>



  </link>

  <gazebo reference="chassis">
    <kp>1000000.0</kp>
    <kd>1.0</kd>
    <mu1>0.8</mu1>
    <mu2>0.8</mu2>
    <maxVel>0.0</maxVel>
    <minDepth>0.001</minDepth>
</gazebo>

  <joint name="base_link_joint" type="fixed">
    <origin rpy="0 0 0" xyz="0 0 0" />
    <parent link="base_link" />
    <child link="chassis" />
  </joint>



</robot>
